### PR TITLE
added user profile and checkout

### PR DESCRIPTION
**Pull request by sanket4120**
Added
- user profile page
- address management
- checkout 
- orders & order details page

Files to review:
- components (addressManagement)
- pages (checkout, orders, orderDetails, account)
- reducers (userReducer)

### CODE REVIEW

File: src/pages/account/account.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@media (max-width: 768px) {
+  .account-nav {
+    /* flex-direction: row;
+    justify-content: space-between;
+    flex-wrap: wrap; */
+  }
```
Comment (sri-rishi) [inline src/pages/account/account.css:6]: Suggestion: if not needed remove comments

File: src/pages/orderDetails/OrderDetails.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                  {orderDetails?.deliveryDetails?.firstName}{' '}
+                  {orderDetails?.deliveryDetails?.lastName}
+                </p>
+                <p className='txt-capitalize'>
+                  {orderDetails?.deliveryDetails?.addressLine1},{' '}
+                  {orderDetails?.deliveryDetails?.addressLine2},
+                  {orderDetails?.deliveryDetails?.city}
+                  {' - '}
+                  {orderDetails?.deliveryDetails?.pincode},{' '}
+                  {orderDetails?.deliveryDetails?.state}
```
Comment (sri-rishi) [inline src/pages/orderDetails/OrderDetails.js:84]: Suggestion: you can use template literals  here

**Review by sri-rishi — APPROVED** [review]
code looks good, functionality is working, feel free to merge

**Review by ShrutiLakhani — APPROVED** [review]
Well structured code. Looks good to merge.

**Pull request merged**